types/jest/index.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for Jest 24.0
-// Project: https://jestjs.io
-// Definitions by: Asana <https://asana.com>
+// Project: https://jestjs.io/
+// Definitions by: Asana (https://asana.com)
```

Comment:
Because [the&nbsp;`dt‑header` rule](https://github.com/microsoft/dtslint/blob/master/docs/dt-header.md) told me&nbsp;to.

Reply:
There is not a single mention of listing it as `Definitions by: My Name (My not github URL)` on there. It all says `Definitions by: My Name <https://github.com/myname>`. The only thing it says is that it is ***preferred*** to list a GitHub URL but preferences are never rules and it is not your place to correct the URL someone else put in the file. And even then it should still be `<https://asana.com>`.

Reply:
Oh you mean when you ran `npm run lint jest`? If so I guess they changed that check since I PR'd for Jest because it didn't say anything about it for me. Alright I guess.